networkx/algorithms/polynomials/tutte.py

```diff
@@ -0,0 +1,91 @@
+"""Functions supporting the computation of the Tutte polynomial of a graph."""
```

Comment:
You should write a concise one paragraph description of what a Tutte polynomial is. It can appear in the docs for this module, and if appropriate also in the doc_string of every appropriate function. Remember you reader is likely to not know what this is, and even if they do, they may have a different definition of it. 

The doc_strings should start with a short one-line description (that gets used to label any links in the docs to that function). It should be short -- like 80 chars or less) This must be followed by a blank line. Then put a description of what the function does in paragraph form. Define terms, give formulas, etc. Make it clear who should use this and perhaps more importantly who shouldn't use it.

You should also include references to articles or books or wikipedia with more info for the interested reader. Do you have publications where this subject is discussed?

Reply:
Good morning @dschult -  these comments make sense, and I've attempted to address them in my most recent commit. 

In particular, I provide an overview of the Tutte polynomial, some specializations and evaluations, three equivalent definitions, brief practical usage, and a few references for further reading.

Thank you for your help!